tools/data/ava/preparing_ava.md

```diff
@@ -0,0 +1,131 @@
+# Preparing AVA
+
+For basic dataset information, please refer to the official [website](https://research.google.com/ava/index.html).
+Before we start, please make sure that the directory is located at `$MMACTION2/tools/data/ava/`.
+
+## Step 1. Prepare Annotations
+
+First of all, you can run the following script to prepare annotations.
+
+```shell
+bash download_annotations.sh
+```
+
+This command will download `ava_v2.1.zip` for AVA `v2.1` annotation. If you need the AVA `v2.2` annotation, you can try the following script.
+
+```shell
+VERSIO=2.2 bash download_annotations.sh
+```
+
+## Step 2. Prepare Videos
+
+Then, use the following script to prepare videos. The codes are adapted from the [official crawler](https://github.com/cvdfoundation/ava-dataset).
+Note that this might take a long time.
+
+```shell
+bash download_videos.sh
+```
+
+Note that if you happen to have sudoer or have [GNU parallel](https://www.gnu.org/software/parallel/) on your machine,
+you can speed up the procedure by downloading in parallel.
```

Comment:
What about wrapping parallel downloading in a python script? I have checked that `parallel` is not a default package on Linux. Seems other data tools in the repo use python to implement parallel downloading.

Reply:
Agree.

Reply:
Done